src/main/java/org/synyx/urlaubsverwaltung/application/application/ApplicationMailService.java

```diff
@@ -86,11 +86,21 @@ void sendAllowedNotification(Application application, ApplicationComment applica
             .build();
         mailService.send(mailToApplicant);
 
+        // Inform all person of interest like boss or department head that the application for leave has been allowed
+        final List<Person> relevantRecipientsToInform = applicationRecipientService.getRecipientsOfInterest(application);
+        final Mail mailToRelevantRecipients = Mail.builder()
+            .withRecipient(relevantRecipientsToInform)
+            .withSubject("subject.application.allowed.boss", application.getPerson().getNiceName())
+            .withTemplate("allowed_office_boss", model)
```

Comment:
Soll `BOSS` wirklich den Kommentar von der Freigabe sehen können?

Oder sollen wir `BOSS` eine Information senden a la:

```
Hallo BOSS,

FREIGABE_PERSON hat folgende Abwesenheit von ABWESEND_PERSON genehmigt.
```

Reply:
warum darf er den Kommentar nicht sehen? Er sieht ihn ja auch wenn in den Antrag geklickt word.